### PR TITLE
create event page for AWS Summit London

### PR DESCRIPTION
**Pull request by shughes26**
## Description
Event page created for AWS Summit London (url `aws-summit`)

## Checklist:
- [x] I have reviewed the [style guide](https://github.com/pulumi/pulumi-hugo/blob/master/STYLE-GUIDE.md).
- [x] If blogging, I have reviewed the [blogging guide](https://github.com/pulumi/pulumi-hugo/blob/master/BLOGGING.md).
- [x] I have manually confirmed that all new links work.
- [x] I added aliases (i.e., redirects) for all filename changes.
- [x] If making css changes, I rebuilt the bundle.

### CODE REVIEW

**Comment by SaraDPH** [comment]
On the templates:

- Container - remove other clouds from the description and link to https://www.pulumi.com/templates/container-service/aws/
- Replace the middle with "Kubernetes Cluster on AWS" description "Deploy a Kubernetes cluster on AWS with Pulumi and Amazon EKS." [this is the image](https://www.pulumi.com/templates/kubernetes/aws/architecture.png) and then link to https://www.pulumi.com/templates/kubernetes/
- remove the other clouds from the description and link to https://www.pulumi.com/templates/serverless-application/aws/

image

File: themes/default/content/aws-summit/_index.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+        date: May 16, 2024
+        description: Learn advanced topics that make up a robust infrastructure CI/CD pipeline through guided exercises.
+        link: /resources/advanced-cicd-aws-pulumi-github-actions/
+        action: Register Now
+
+templates:
+    items:
+        - heading: Container Service Templates
+          description: Pulumi program templates are the fastest way to deploy container services on AWS.
+          image: /images/kubecon/templates-container-service-aws.png
```
Comment (aaronkao) [inline themes/default/content/aws-summit/_index.md:33]: why don't you just link it to the origin file - /templates/container-service/aws/architecture.png

File: themes/default/content/aws-summit/_index.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+          action: Try it
+          link: /templates/container-service/aws
+        - heading: Kubernetes Cluster on AWS
+          description: Deploy a Kubernetes cluster on AWS with Pulumi and Amazon EKS.
+          image: /templates/kubernetes/aws/architecture.png
+          action: Try it
+          link: /templates/kubernetes/aws
+        - heading: Serverless Templates
+          description: Easily deploy serverless applications on AWS with Pulumi Serverless Application templates.
+          image: /images/kubecon/templates-serverless-application.png
```
Comment (aaronkao) [inline themes/default/content/aws-summit/_index.md:43]: same with this one templates/serverless-application/aws/architecture.png

File: themes/default/content/aws-summit/_index.md [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+      - title: Advanced CI/CD for AWS using Pulumi and GitHub Actions
+        date: May 16, 2024
+        description: Learn advanced topics that make up a robust infrastructure CI/CD pipeline through guided exercises.
+        link: /resources/advanced-cicd-aws-pulumi-github-actions/
+        action: Register Now
+
+templates:
+    items:
+        - heading: Container Service Templates
+          description: Pulumi program templates are the fastest way to deploy container services on AWS.
```
Comment (aaronkao) [inline themes/default/content/aws-summit/_index.md:32]:
```suggestion
          description: Deploy a container service on AWS with Pulumi and Amazon ECS.

File: themes/default/content/aws-summit/_index.md [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+          image: /images/kubecon/templates-container-service-aws.png
+          action: Try it
+          link: /templates/container-service/aws
+        - heading: Kubernetes Cluster on AWS
+          description: Deploy a Kubernetes cluster on AWS with Pulumi and Amazon EKS.
+          image: /templates/kubernetes/aws/architecture.png
+          action: Try it
+          link: /templates/kubernetes/aws
+        - heading: Serverless Templates
+          description: Easily deploy serverless applications on AWS with Pulumi Serverless Application templates.
```
Comment (aaronkao) [inline themes/default/content/aws-summit/_index.md:42]:
```suggestion
          description: Deploy a serverless application on AWS with Pulumi, AWS Lambda, and Amazon API Gateway.
```

**Review by aaronkao — APPROVED** [review]
Made some copy updates. Otherwise looks good to go. Thanks

**Pull request merged**